examples/cephfs/storageclass.yaml

```diff
@@ -14,11 +14,11 @@ parameters:
   # represent the Ceph cluster in clusterID below
   clusterID: <cluster-id>
 
-  # CephFS filesystem name into which the volume shall be created
-  fsName: myfs
+  # (required) CephFS filesystem name into which the volume shall be created
```

Comment:
Since we are getting rid of default values, Adding examples might make the job easier
For example: 
```
# (required) CephFS filesystem name into which the volume shall be created
# eg: fsName: myfs
fsName: <cephfs-name>
```
Thoughts?

Reply:
@pkalever are you planning to add examples?

Reply:
All required and optional options are mentioned in doc https://github.com/ceph/ceph-csi/blob/master/docs/deploy-rbd.md

Reply:
[Excuse me, I missed this update somehow]
It's easy to understand what goes in those fields, but again nothing wrong in adding the example, yes I can do that :-)
Thanks all.